.github/workflows/docker.yml

```diff
@@ -13,8 +13,12 @@ defaults:
   run:
     shell: bash
 
+env:
```

Comment:
Is this whole commit about docker related to this PR or can it be merged separately?